packages/turf-invariant/index.js

```diff
@@ -195,6 +195,34 @@ function getGeomType(geojson) {
     if (geom) return geom.type;
 }
 
+/**
+ * Get GeoJSON object's type, Geometry type is prioritize.
+ *
+ * @param {GeoJSON} geojson GeoJSON object
+ * @param {string} [name] of the variable
+ * @returns {string} GeoJSON type
+ * @example
+ * var point = {
+ *   "type": "Feature",
+ *   "properties": {},
+ *   "geometry": {
+ *     "type": "Point",
+ *     "coordinates": [110, 40]
+ *   }
+ * }
+ * var geom = turf.getType(point)
+ * //="Point"
+ */
+function getType(geojson, name) {
+    if (!geojson) throw new Error('geojson is required');
```

Comment:
Also here it should probably use `name` if available. Like maybe:
```js
if (!geojson) throw new Error((name || 'geojson') + ' is required');
```

Reply:
That looks good to me!